packages/expo-standard-web-crypto/src/getRandomValues.ts

```diff
@@ -21,7 +21,7 @@ type IntegerArrayConstructor =
   | Uint8ClampedArrayConstructor;
 
 /**
- * An implementation of Crypto.getRandomValues that uses expo-random's secure random generator if
+ * An implementation of Crypto.getRandomValues that uses expo-crypto's secure random generator if
```

Comment:
The `getRandomValues` implementation in this file could be changed to use `expo-crypto`'s `getRandomValues` instead. Not sure if it's going to cause issues.